sources/error/error.go

```diff
@@ -4,34 +4,34 @@ import (
 	"gopkg.in/Clever/optimus.v3"
 )
 
-// ErrorTable implemements an Optimus Table
+// Table implemements an Optimus Table
 // It's purpose is to return a given error
-type ErrorTable struct {
```

Comment:
Need to revert this before merging, and fix the Makefile to not test this pkg strictly